dftimewolf/lib/processors/turbinia_gcp_tam.py

```diff
@@ -219,22 +220,38 @@ def TurbiniaProcess(
       self.client.send_request(request)
       self.logger.info('Waiting for Turbinia request {0:s} to complete'.format(
           request.request_id))
-      self.client.wait_for_request(**request_dict)
-      task_data = self.client.get_task_data(**request_dict)
+
+      # Workaround for rate limiting in turbinia when checking task status
+      finished = False
+      while not finished:
+        try:
+          self.client.wait_for_request(**request_dict)
+          task_data = self.client.get_task_data(**request_dict)
+
+          message = self.client.format_task_status(
+              full_report=True, **request_dict)
+          short_message = self.client.format_task_status(**request_dict)
+          self.logger.info(short_message)
+
+          # Store the message for consumption by any reporting modules.
+          report = containers.Report(
+              module_name='TurbiniaProcessor',
+              text=message,
+              text_format='markdown')
+          self.state.StoreContainer(report)
+
+          finished = True
+        except Exception as exception: # pylint: disable=broad-except
```

Comment:
What exception is actually getting thrown here, is it a TurbiniaException?  Can we scope this more narrowly?

Hopefully this will fix this, I just need to push a new release: https://github.com/google/turbinia/pull/902/files

Reply:
No, it's a `RuntimeError`, and no, that new turbinia release doesn't look to have fixed the issue.